zulipterminal/helper.py

```diff
@@ -525,7 +525,8 @@ def notify(title: str, html_text: str) -> None:
     elif MACOS:  # NOTE Tested and should work!
         command = (
             "osascript -e "
-            "'display notification \"\'{}\'\" with title \"\'{}\'\"'"
+            "'display notification \"\'{}\'\" with title \"\'{}\'\" "
+            " sound name \"ZT_NOTIFICATION_SOUND\"'"
```

Comment:
I think the environment variable approach here is better for the main reason that this is needed only for the OSX platform. Having it in the config file would 
a) require quite some work to parse it (from file and maybe have a tag)
b) it would really be only specific to OSX so it wouldn't make much sense to keep it in conf file.

Hence, I went with this much simpler approach of having an env variable.

Reply:
This definitely seems fine for now, I just wasn't sure if the default here was from an environment variable, in which case that's fine - certainly for v1 :+1: